### ISSUE TITLE
Give user an indicator when current field input is expandable

### ISSUE THREAD

**Issue by stardiviner** [capitaomorte/yasnippet#690]
For example, I have two snippet: `template`, `temp`
When I expand snippet `template`, and type in `temp` in field 1 of `template`. I hope the cursor color can change, how to do this?

**Comment by npostavs** [comment]
hmm, maybe something could be added to the post command hook?

**Comment by stardiviner** [comment]
Yeah, if it is the best solution for now. I checked my `post-command-hook`.
```lisp
(company-post-command yas--post-command-handler t flyspell-post-command-hook)
```
